SPEC-annex-A-filesystem-storage.md

```diff
@@ -16,16 +16,19 @@ Example:
 A build tool should choose to persist a mapping between artifacts and their corresponding GitBOM documents.  If it chooses
 to do so it should for each artifact persist a symlink:
 
-```${GITBOM_DIR}/a2g/${Artifact Identifier Type uri prefix with ':' replaced by '_'}/${artifact id}:0:2}/${artifact id:2:} -> ${relative symlink to GitBOM document for artifact}```
+```${GITBOM_DIR}/a2g/${Artifact Identifier Type uri prefix with ':' replaced by '_'}/${artifact id}:0:2}/${artifact id:2:}/${GitBOMID:0:2}/${GitBOMID:2:} -> ${relative symlink to GitBOM document for artifact}```
 
 `a2g` is short for `artifact to graph`.
 
 Example:
 
 ```
-.adg/a2g/gitoid_blob_sha1/0e/8efd4cdf0d5bafcfcae658c2662a73b199b301 -> ../../../../../objects/gitoid/blob/sha1/1d/6e79da5e380e5d3e5adcf899c4d65c0e80bfb3
+.adg/a2g/gitoid_blob_sha1/0e/8efd4cdf0d5bafcfcae658c2662a73b199b301/6e/79da5e380e5d3e5adcf899c4d65c0e80bfb3 -> ../../../../../objects/gitoid_blob_sha1/1d/6e79da5e380e5d3e5adcf899c4d65c0e80bfb3
 ```
 
+In the event that GITBOM_NO_EMBED is in use, the artifact to GitBOM Document mapping can be used to identify the GitBOM ID for the artifact as an alternative to reading it from the artifact itself (embedded mode).
+If more than one GitBOM ID is indicated in the artifact to graph mapping for a given artifact, the lexically first GitBOM ID should be used.
```

Comment:
I think it is better to use the chronically latest gitBOM doc, instead of the lexically first gitBOM doc.